modules/farm_fd2_school/src/entrypoints/FD1/App.vue

```diff
@@ -176,6 +166,29 @@ export default {
       this.unit = null;
       this.comment = '';
     },
+    async fetchCrops() {
+      const cropsResponse = await fetch(
+        'http://farmos/api/taxonomy_term/plant_type'
+      );
+      const crops = await cropsResponse.json();
+      this.cropList = crops.data;
+    },
+    async fetchPlantList(cropURL) {
+      const plantResponse = await fetch(cropURL);
+      const plants = await plantResponse.json();
+      this.plantList = plants;
+    },
+  },
+  watch: {
+    crop() {
```

Comment:
- Because `await` is (or should be) used in this method or hook it needs to be declared as `async`.